lutris/sysoptions.py

```diff
@@ -161,6 +161,34 @@ def get_optirun_choices():
         'advanced': True,
         'help': "Environment variables loaded at run time"
     },
+    {
+        'option': 'play_script',
+        'type': 'file',
+        'label': 'Play Script',
+        'advanced': True,
+        'help': ("Choose a script to execute with script icon")
+    },
+    {
+        'option': 'pre_script',
+        'type': 'file',
+        'label': 'Pre Launch Script',
+        'advanced': True,
+        'help': ("Choose a script to execute at game start")
+    },
+    {
+        'option': 'wait_for_script_completion',
+        'label': "Wait for script completion before starting the game",
```

Comment:
WTF

Reply:
this is a tick box to use with pre_launch_script basically when this is checked the game will only start after the pre_launch_script finishes its work, this works by using the script thread join() fn,
as opposed to the default behavior which is the game can be started even if the pre_launch_script is still running